### DISCUSSION TITLE
Friends interconnectedness scheme

### DISCUSSION THREAD

**Discussion by GWPvGWP** [macedonga/beunblurred#77] [Ideas]
Project a diagram that shows connections between friends. 

Idea taken from: https://shomil.me/bereal/

**Comment by macedonga** [comment]
It would be really cool, but the endpoint `https://us-central1-alexisbarreyat-bereal.cloudfunctions.net/friendSuggestion` doesn't exist anymore, and neither `https://mobile.bereal.com/api/friendSuggestion` or `https://mobile.bereal.com/api/friendSuggestions`...

I tried looking in the apk for any endpoint that looks like it would do that but didn't find anything... BeReal probably removed that endpoint after that blog post released :-(

**Comment by GWPvGWP** [comment]
I was thinking of a slight adaptation, using the 'common friends' of only your own friends. Since common friends are in BeUnblurred yet it should be possible, at least theoretically?
